nemo/lightning/pytorch/strategies/utils.py

```diff
@@ -21,6 +22,14 @@
 from nemo.utils.callbacks.dist_ckpt_io import AsyncFinalizableCheckpointIO
 
 
+@dataclass(kw_only=True)
+class SelectiveRestoreConfig:
+    path: str
+    adapter_path: Optional[str] = None
+    model_weights: bool = True
```

Comment:
How about `load_model_state` & `load_optim_state`?

I want to avoid a verb here but also make sure the options aren't confused with the actual thing.

Reply:
Yeah I'm fine with the rename, will do.

Reply:
thank you.